Sources/main.swift

```diff
@@ -1,11 +1,24 @@
 import Commander
 import PathKit
 
+private func target_specific_arguments() throws -> [String] {
+#if os(OSX)
+  return [
+    "-sdk",
+    "/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.sdk",
+    "-target", "x86_64-apple-macosx10.10",
```

Comment:
SPM is going to get test support sometime. So perhaps it isn't worth investing time in it.

Will merge this for the interim.